jdaviz/configs/default/plugins/model_fitting/tests/test_plugin.py

```diff
@@ -14,7 +14,7 @@
 
 
 def test_default_model_labels(specviz_helper, spectrum1d):
-    specviz_helper.load_data(spectrum1d)
+    specviz_helper.load_spectrum(spectrum1d)
```

Comment:
does `load_data` skip some important logic introduced in this PR?  I'm wondering if we need to make it an alias to `load_spectrum` (and perhaps deprecate `load_spectrum`) since the distinction is quite confusing for the user (and for devs)... although that probably could/should be a follow-up PR.